libs/tup-components/src/projects/details/ProjectDetails.tsx

```diff
@@ -130,6 +131,30 @@ const Grant: React.FC<{
   );
 };
 
+const NewAllocation = () => (
+  <>
+    <a href="https://submit-tacc.xras.org/" target="_blank" rel="noreferrer">
+      <Button className={styles['tap-button']} type="link">
```

Comment:
Do the `tap-button` style[^1] have any effect here (or the other added instance in this PR)? If not, please remove.

[^1]: It seems to [come from and be for ManageAccount](https://github.com/TACC/tup-ui/search?q=tap-button)